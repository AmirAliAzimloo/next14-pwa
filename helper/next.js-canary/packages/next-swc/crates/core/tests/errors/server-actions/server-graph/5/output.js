/* __next_internal_action_entry_do_not_use__ {} */ import { createActionProxy } from "private-next-rsc-action-proxy";
import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
export * from 'foo';
import { ensureServerEntryExports } from "private-next-rsc-action-validate";
ensureServerEntryExports([]);
