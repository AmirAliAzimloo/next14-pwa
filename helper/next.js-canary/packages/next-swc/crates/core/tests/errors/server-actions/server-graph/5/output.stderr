
  x Only async functions are allowed to be exported in a "use server" file.
   ,-[input.js:2:1]
 2 | 
 3 | export * from 'foo'
   : ^^^^^^^^^^^^^^^^^^^
   `----
